### PR TITLE
CVE-2015-3183

### PR DESCRIPTION
**Pull request by Lixnir**

### CODE REVIEW

File: cves/CVE-2015-3183.yml [hunk 1]
```diff
@@ -4,36 +4,36 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
-CVSS: AV:N/AC:L/Au:N/C:N/I:P/A:N
+CWE: 444
+CVSS: 7.3 (CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L) - was AV:N/AC:L/Au:N/C:N/I:P/A:N
```
Comment (andymeneely) [inline cves/CVE-2015-3183.yml]: You can ignore this part - we bring it in automatically

Comment (andymeneely) [reply cves/CVE-2015-3183.yml]: (and we'll bring in the v3 ones later)

File: cves/CVE-2015-3183.yml [hunk 2]
```diff
@@ -66,17 +68,17 @@ fixes_vcc_instructions: |
...
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 29779fd08c18b18efc5e640d74cbe297c7ec007e
-     note:
-   - commit: c3f98ce69b9c8ddc936380e9d7cf03feca809558
-     note:
-   - commit: 87fcef685c19652353c2e26e90ed1a2aa8099629
-     note:
+  - commit: 29779fd08c18b18efc5e640d74cbe297c7ec007e
+    note: Cleaned up how the system parses requests
```
Comment (devonbagley97) [inline cves/CVE-2015-3183.yml]: Do you have any specifics on how the parsing was cleaned up? I'm curious about what kind of changes were made that fixed the http smuggling. Could potentially add some of that detail here or in the description for future reference.

File: cves/CVE-2015-3183.yml [hunk 3]
```diff
@@ -155,12 +156,12 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: I didn't see any significant events based on the relative research I found.
```
Comment (bnk5096) [inline cves/CVE-2015-3183.yml]: Perhaps "relative" should become "related", I am not the best at grammar, but i think this makes a bit more sense.

File: cves/CVE-2015-3183.yml [hunk 4]
```diff
@@ -66,17 +68,17 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 29779fd08c18b18efc5e640d74cbe297c7ec007e
-     note:
-   - commit: c3f98ce69b9c8ddc936380e9d7cf03feca809558
-     note:
-   - commit: 87fcef685c19652353c2e26e90ed1a2aa8099629
-     note:
+  - commit: 29779fd08c18b18efc5e640d74cbe297c7ec007e
```
Comment (bnk5096) [inline cves/CVE-2015-3183.yml:71]: In a quick search through the git commits, I couldn't locate this hash outside of being mentioned in another issue on github. Maybe double check this commit hash to make sure this is right.

Comment (Lixnir) [reply cves/CVE-2015-3183.yml:71]: This is one of multiple hashes I found that for some reason don't show up when you search for them, but if you manually change the URL it shows up: https://github.com/apache/httpd/commit/29779fd08c18b18efc5e640d74cbe297c7ec007e

**Pull request merged**